apps/files/src/components/FilesAppBar.vue

```diff
@@ -583,6 +606,22 @@ export default {
       }
       this.resetFileSelection()
       this.setHighlightedFile(null)
+    },
+
+    moveResources() {
+      const resources = JSON.parse(JSON.stringify(this.selectedFiles))
```

Comment:
strange gymnastics ? is that a trick to do a deep copy ?
would it make sense to only gather the file ids ?

Reply:
This copies only the values without the watchers, etc. related to the vuex state. By doing this we are safe that we don't get any errors later on in case of modifying the values + we do not modify the original state. It looks a bit odd but I haven't yet found in any vue issue/forum different way to do this. Well, I saw one with lodash but I'd prefer to avoid that 🤷